met-api/src/met_api/services/submission_service.py

```diff
@@ -140,6 +152,75 @@ def validate_review(cls, values: dict, user):
            not rejected_reason_other:
             raise ValueError('A rejection reason is required.')
 
+    @classmethod
+    def add_or_update_staff_note(cls, survey_id, submission_id, staff_notes):
+        """Process staff note for a comment."""
+        for staff_note in staff_notes:
+            note = StaffNote.get_staff_note_by_type(submission_id, staff_note.get('note_type'))
+            if note:
+                note.note = staff_note['note']
+                StaffNote.flush(note)
```

Comment:
flush a save method from the basemodel right? I think its the fist time i've seen it in our repo so i'm confused